Assets/Scripts/Room/PlayerManager.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Photon.Pun;
+
+public class PlayerManager : MonoBehaviour
+{
+    private readonly int deadPlayerLimit = 2;
+
+    private static PlayerManager _instance;
+    public static PlayerManager Instance { get => _instance; }
+
+    [SerializeField] private GameObject deathScreen;
+    [SerializeField] private PhotonView photonView;
+
+    private int deadPlayerCount = 0;
+
+    void Awake()
+    {
+        // singleton
+        if (_instance != null && _instance != this)
+        {
+            Destroy(gameObject);
+        }
+        else
+        {
+            _instance = this;
+        }
+        DontDestroyOnLoad(gameObject);
```

Comment:
Might want to remove `DontDestroyOnLoad` as we probably don't want the `PlayerManager` to follow the player into the menus. Removing the line would also allow it to refresh for each level.